GrandCentralBoard/Widgets/Watch/Server/Event.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  EventModel.swift
+//  GrandCentralBoard
+//
+//  Created by Michał Laskowski on 07.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import Decodable
+
+struct DateFormattingError : ErrorType {
+    let dateString: String
+}
+
+struct Event : Decodable {
+
+    private static let dateFormatter: NSDateFormatter =  {
+        let formatter = NSDateFormatter()
+        formatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZ";
```

Comment:
This logic has a bug. In order to properly use date formatter to parse network data you have to set its `locale` property. Follow https://developer.apple.com/library/ios/qa/qa1480/_index.html for further explanation.